R/calcpm_delta_kap_industry.R

```diff
@@ -0,0 +1,31 @@
+#' Calculate depreciation rate of industry energy-efficiency capital (EEK).
+#' 
+#' @md
+#' @return A list with a [`magpie`][magclass::magclass] object `x`, `unit`, and
+#'   `description`.
+#'   
+#' @author Michaja Pehl
+#'
+#' @seealso [`calcOutput()`]
+#' 
+#' @importFrom magclass new.magpie
+#' 
+
+#' @export
+calcpm_delta_kap_industry <- function() {
```

Comment:
is this a calc* function? or a tool* function?

Reply:
It is a "calc" function.

Reply:
ah, I see....